packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vdsm/tasks/configure.yml

```diff
@@ -75,7 +75,13 @@
      allow perm=any trust=1 : dir=/etc/vdsm/mom.d/ ftype=text/x-lisp
   when: ansible_distribution == 'RedHat'
 
+- name: collect facts about system services
+  service_facts:
+
 - name: restart fapolicy service
   systemd:
    state: restarted
    name: fapolicyd
+  when:
+    - "'fapolicyd.service in services"
```

Comment:
What's the "'" character for?

Reply:
missed character for service name, fixed